packages/nimble-components/.storybook/main.js

```diff
@@ -22,12 +24,14 @@ export const addons = [
             }
         }
     },
-    '@storybook/addon-a11y',
-    '@storybook/addon-interactions'
+    getAbsolutePath('@storybook/addon-a11y'),
+    getAbsolutePath('@storybook/addon-interactions'),
+    getAbsolutePath('@chromatic-com/storybook'),
```

Comment:
I sunk the changes down, did an npm install + build + `npm run storybook -w pacakges\nimble-components` to try out this plugin locally but I get this error:
![image](https://github.com/ni/nimble/assets/1588923/063ede31-de69-44f4-89bf-f8df42816835)

Reply:
Looks like they added an oddly documented config requirement… fixed now